anta/tests/routing/bgp.py

```diff
@@ -447,3 +447,103 @@ def test(self) -> None:
 
         if failures:
             self.result.is_failure(f"Failures: {list(failures.values())}")
+
+
+class VerifyBGPCapabilities(AntaTest):
```

Comment:
I think we should split in multiple tests, each test being a specific capability we want to test. It will be easier to manage and easier to understand for the users:

`VerifyBGPNeighborMPCaps` --> With this one the user can provide a list of MP capabilities to check
`VerifyBGPNeighborASNCap` --> This one doesn't need any input
`VerifyBGPNeighborRouteRefreshCap` --> Same thing, no input required

Reply:
We can add more in the future if required.

Reply:
we still need to add neighbor and vrf as user input.